corehq/apps/userreports/tasks.py

```diff
@@ -34,7 +34,7 @@ def _build_indicators(config, document_store, relevant_ids, resume_helper):
 
 
 @task(queue='ucr_queue', ignore_result=True)
-def rebuild_indicators(indicator_config_id, initiated_by=None):
+def rebuild_indicators(indicator_config_id, initiated_by=None, count=-1, offset=0):
```

Comment:
will we ever need an offset other than 0? Might keep things cleaner to get rid of that parameter unless we know we'll use it.